light-zk.js/src/wallet/user.ts

```diff
@@ -987,9 +983,10 @@ export class User {
     // we send an array of instructions to the relayer and the relayer sends 3 transaction
     const txHash = await this.sendTransaction();
 
-    var relayerMerkleTreeUpdateResponse = "notPinged";
+    let relayerMerkleTreeUpdateResponse = "notPinged";
 
     if (confirmOptions === ConfirmOptions.finalized) {
+      // Don't add await here
```

Comment:
```suggestion
      // Don't add await here, because the utxos have been spent the transaction is final. We just want to ping the relayer to update the Merkle tree not wait for the update. This option should be used to speed up transactions when we do not expect a following transaction which depends on the newly created utxos.
```